src/export/InstanceExporter.ts

```diff
@@ -432,14 +432,13 @@ export class InstanceExporter implements Fishable {
     }
 
     const instanceOfStructureDefinition = StructureDefinition.fromJSON(json);
-
     let instanceDef = new InstanceDefinition();
     instanceDef._instanceMeta.name = fshDefinition.id; // This is name of the instance in the FSH
     if (fshDefinition.title) {
-      instanceDef._instanceMeta.title = fshDefinition.title;
+      instanceDef.title = fshDefinition.title;
```

Comment:
I have a few concerns about this implementation:
* The title should only be assigned on instances that have usage `definition`.
* The title should only be assigned on conformance instances that support a `title` element.  For example, the `Patient` resource does not have a title element, but this code would set one anyway.
* Existing code still depends on `instanceDef._instanceMeta.title`, but that's been removed here -- so that existing code won't work right anymore.

Reply:
Fixed.